resources/visualizations/Top-10-Attachment-Types-By-Count.json

```diff
@@ -0,0 +1,9 @@
+{
+  "title": "Top 10 Attachment Types By Count",
```

Comment:
@Rob, would we be interested for (3.1.2. to get "Bottom 10 Attachment Types By Count"?